api/ssrApollo.js

```diff
@@ -160,6 +175,18 @@ export function getGetServerSideProps (
       me = null
     }
 
+    // Check if the user account is deleted
+    if (me) {
+      const user = await models.user.findUnique({
+        where: { id: parseInt(me.id) }, // Convert string to int
+        select: { deletedAt: true }
+      })
+
+      if (user?.deletedAt) {
+        me = null
+      }
+    }
+
```

Comment:
Is this needed? Aren't you already doing this in `getSSRApolloClient` so using that client to query `ME` will return null?

Reply:
Not sure, will have a look.